newIDE/app/src/MainFrame/index.js

```diff
@@ -1703,7 +1703,6 @@ class MainFrame extends React.Component<Props, State> {
       this.state.loadingProject ||
       this.state.previewLoading ||
       this.props.loading;
-
```

Comment:
Same, re-read every line and cancel all changes not needed to have a PR as minimal as possible